domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/fpinntektsmelding/OpprettForespørselResponsNy.java

```diff
@@ -5,7 +5,7 @@
 import jakarta.validation.Valid;
 import jakarta.validation.constraints.NotNull;
 
-public record OpprettForespørselResponsNy(@NotNull @Valid List<OrganisasjonsnummerMedStatus> organisasjonsnumreMedStatus) {
+public record OpprettForespørselResponsNy(@NotNull List<@Valid OrganisasjonsnummerMedStatus> organisasjonsnumreMedStatus) {
```

Comment:
Hva tenker man om

`@NotNull List<@Valid @NotNull OrganisasjonsnummerMedStatus> `

For å forsikre seg om at elementene i listen ikke kan være null?

Reply:
Jeg tror Nullable - sjekking er et eget testområde. Går mer på semantikk enn å sikre inputvalidering